src/main/java/net/minecraftforge/fml/common/Loader.java

```diff
@@ -680,78 +682,110 @@ public ModClassLoader getModClassLoader()
         return modClassLoader;
     }
 
-    public void computeDependencies(String dependencyString, Set<ArtifactVersion> requirements, List<ArtifactVersion> dependencies, List<ArtifactVersion> dependants)
+    public static void computeDependencies(String dependencyString, Set<ArtifactVersion> requirements, List<ArtifactVersion> dependencies, List<ArtifactVersion> dependants)
     {
         if (dependencyString == null || dependencyString.length() == 0)
```

Comment:
with all of these suggestions: Only if you want to, it's not a terrible suggestion since you're rewriting most of this anyway, but it's certainly not required.